projects/demo/src/modules/components/textfield/examples/4/index.ts

```diff
@@ -0,0 +1,25 @@
+import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {MaskitoDirective} from '@maskito/angular';
+import {maskitoNumberOptionsGenerator} from '@maskito/kit';
+import {TuiIcon, TuiTextfield} from '@taiga-ui/core';
+import {TuiTooltip} from '@taiga-ui/kit';
+
+@Component({
+    standalone: true,
+    imports: [FormsModule, MaskitoDirective, TuiIcon, TuiTextfield, TuiTooltip],
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    encapsulation,
+    changeDetection,
+})
+export default class Example {
+    protected value = Math.PI.toFixed(48);
+    protected readonly options = maskitoNumberOptionsGenerator({
+        decimalSeparator: ', ',
```

Comment:
decimal separator should not be more than 1 character, I think that's the problem with the example

Reply:
fixed